src/app/pages/splash/services/splash.service.ts

```diff
@@ -2,12 +2,15 @@ import { Injectable } from '@angular/core'
 
 import { StorageService } from '../../../core/services/storage.service'
 import { StorageKeys } from '../../../shared/enums/storage'
+import { AuthService } from '../../auth/services/auth.service'
 
 @Injectable()
 export class SplashService {
-  constructor(public storage: StorageService) {}
+  constructor(public storage: StorageService, private auth: AuthService) {}
 
   evalEnrolment() {
-    return this.storage.get(StorageKeys.PARTICIPANTLOGIN)
+    return this.auth.isRefreshTokenExpired().then(expired => {
+      return expired ? false : this.storage.get(StorageKeys.PARTICIPANTLOGIN)
```

Comment:
what will happen here if there is a new enrolment and there is not token in the storage ? 
I think the first check should be for participant login then check for token expiry

Reply:
Okay. I have made the changes.